tests/test.py

```diff
@@ -17,6 +17,14 @@ def test_datadep() -> None:
     assert data_dep.is_file(), f"{data_dep} does not exist"
     assert (size := data_dep.stat().st_size) == 591, f"{data_dep} has wrong size {size}"
 
+    data_dep = Path("tests/appimage_data_file.txt")
+    assert data_dep.is_file(), f"{data_dep} does not exist"
+    assert (size := data_dep.stat().st_size) == 13, f"{data_dep} has wrong size {size}"
```

Comment:
That's a little obscure, tbh I am not sure anymore why I tested the size of these files. Probably to rule out that they're symlinks, and the right file?

Reply:
Yeah I guess it's important to make sure that they're not symlinks into the source/bazel cache. Was that ever a problem?

Reply:
Probably yes. In a previous code base I worked on there were so many edge cases, and I hit them all :D